.travis.yml

```diff
@@ -1,5 +1,10 @@
 sudo: false
 language: rust
+
+skip_commits:
```

Comment:
Are you sure that this is supported on Travis? I wasn't able to find any reference to this. If it isn't supported, then you'll have to do something like https://github.com/facebook/react/pull/2000/files.